src/apps/search-header/search-header.tsx

```diff
@@ -269,7 +276,11 @@ const SearchHeader: React.FC = () => {
       // Before redirecting we need to clean persisted filters from previous search.
       clearFilter();
       redirectTo(
-        constructSearchUrl(searchUrl, determineSuggestionTerm(selectedItem))
+        constructSearchUrlByType(
```

Comment:
Are we introducing a new way of formatting search urls? If yes - do we need to handle backwards compatibility of "old" search urls?